mapswipe-dashboard/app/components/Header/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { memo } from 'react';
+import { _cs } from '@togglecorp/fujs';
```

Comment:
Is this package meant to stay or is it an internal one for development only? There is no README on your github repo, so it's very hard to know what this does.

Reply:
This is meant to stay. We do have a documentation but it's hard to find.
https://github.com/toggle-corp/fujs/blob/develop/mydocs/modules.md